src/components/home/Home.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="">
+    <div class="grid grid-cols-2">
+      <div class="profile">Profile goes here</div>
+      <div class="">Trip setting goes here</div>
```

Comment:
jeżeli to zwykłe niestylowane komponenty to albo usuń dla ob class, albo w obu nadaj roboczą nazwę, żeby nie zostawały puste atrybuty